src/server/rpc/procedures/NASA/NASA.js

```diff
@@ -0,0 +1,65 @@
+// This will utilize NASA's public APIs in order to provide data to the client
+
+'use strict';
+
+var debug = require('debug'),
+	log = debug('NetsBlox:RPCManager:NASA:log'),
+	error = debug('NetsBlox:RPCManager:NASA:error'),
+	trace = debug('NetsBlox:RPCManager:NASA:trace'),
+	request = require('request'),
+	KEY = process.env.NASA_KEY,
+	APOD_URL = 'https://api.nasa.gov/planetary/apod?api_key=' + KEY,
+	MARS_URL = 'http://marsweather.ingenology.com/v1/latest/';
+
+module.exports = {
+
+	isStateless: true,
+	getPath: () => '/NASA',
+	getActions: () => ['apod', 'apodMedia', 'latestMarsWeather'], // list of available functions for client to use
+
+	// NASA's 'Astronomy Picture of the Day'
+	apod: function(req, res) {
+		// to return
+		var results = [];
+
+		request(APOD_URL, function(err, response, body) {
+			body = JSON.parse(body);
+			results.push('Date: ' + body.date);
+			results.push('Title: ' + body.title);
+			results.push('URL: ' + body.url);
+			results.push('Description:');
+			var description = body.explanation.match(/.{1,70}/g) || [];
```

Comment:
I am not sure I like this approach bc the students can change the screen size (as well as font size) and this will no longer be the case. I think it would be better to probably leave it to the user to handle and just return a single text description